> > @smaticka you nedd to fix coding norms and then this one will be good to go.
> 
> @PatNichols do you know how I can see the code norms that need to be changed? for python code, I run the pycode style command. not sure for c++. when I click on the `Details` link in the `Checks` section, I see nothing in the bottom 3 (green checks) and I get access denied to the top 3 (red x's). <img alt="image" width="747" src="https://private-user-images.githubusercontent.com/40580449/300129585-fd9d5804-068f-494a-9a66-e3670227613d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY1NTA2OTYsIm5iZiI6MTcwNjU1MDM5NiwicGF0aCI6Ii80MDU4MDQ0OS8zMDAxMjk1ODUtZmQ5ZDU4MDQtMDY4Zi00OTRhLTlhNjYtZTM2NzAyMjc2MTNkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI5VDE3NDYzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZhMDkzN2UwYzBkNDI5N2M5MTBkMTE5YzNmMmQwNjRmZjY1ZWVmNTgyOTgxOTE4YWQwMDJhZTM2YTgyMTU0MjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zdGldAi8XhDPWQ3gkgxWNK6rNxpAv3k4wrsCw-_e2oc">

@smaticka You can run autopep8 on your code which will change most things to a format that  pycodestyle accepts. Here are the errors if you want to change them by hand.
./cygnss_2ioda.py:52:1: E122 continuation line missing indentation or outdented
./cygnss_2ioda.py:129:33: W291 trailing whitespace
./cygnss_2ioda.py:204:92: E261 at least two spaces before inline comment
./cygnss_2ioda.py:223:1: E302 expected 2 blank lines, found 1

--- a/src/hdf5/cygnss_2ioda.py
+++ b/src/hdf5/cygnss_2ioda.py
@@ -10,8 +10,6 @@
 """
 Python code to ingest netCDF4 CYGNSS data
 """
-import re
-import pdb
 import logging
 import argparse
 from datetime import datetime, timezone
@@ -49,7 +47,6 @@ MetaDataKeyList = [
     ("longitude", "float", "degrees_east"),
     ("dateTime", "long", "seconds since 1970-01-01T00:00:00Z"),
     ("sensorIdentification", "string", ""),
-#    ("stationElevation", "float", "m"),
     ("height", "float", "m"),
 ]
 meta_keys = [m_item[0] for m_item in MetaDataKeyList]
@@ -126,7 +123,7 @@ def main(args):
         # Change time reference
         obs_data_append = adjust_dateTime(obs_data_append, dat_ref)
 
-        # Change longitude range 
+        # Change longitude range
         obs_data_append = adjust_longitude(obs_data_append)
 
         # Append to data frame containing all timestamp data
@@ -201,7 +198,7 @@ def get_data_from_file(afile, col_names):
     longitude = [afile['lon'][ii] for ii in range(len(afile['lon']))]
     dateTime = [int(afile['sample_time'][ii]) for ii in range(len(afile['sample_time']))]  # datetime with different ref time
     windSpeedAt10M = [afile['wind_speed'][ii] for ii in range(len(afile['wind_speed']))]
-    sensorIdentification = [str(afile['sv_num'][ii]) for ii in range(len(afile['sv_num']))] # sv_num is the GPS space vehicle number
+    sensorIdentification = [str(afile['sv_num'][ii]) for ii in range(len(afile['sv_num']))]  # sv_num is the GPS space vehicle number
 
     # Make a column to have a constant elevation for the "station"
     height = [10]*len(dateTime)
@@ -212,13 +209,13 @@ def get_data_from_file(afile, col_names):
 
     # All observation data for this file to append to the master dataframe
     obs_data_append = pd.DataFrame(data_lists, columns=col_names)
-#    pdb.set_trace()
     return obs_data_append
 
 
 def adjust_dateTime(obs_DF, dat_ref):
     obs_DF['dateTime'] = obs_DF['dateTime']+int(dat_ref)
     return obs_DF
+
 
 def adjust_longitude(obs_DF):
     obs_DF['longitude'] = (obs_DF['longitude']+int(180)) % 360 - int(180)

--- a/src/hdf5/cygnss_2ioda.py
+++ b/src/hdf5/cygnss_2ioda.py
@@ -218,7 +218,7 @@ def adjust_dateTime(obs_DF, dat_ref):
 
 
 def adjust_longitude(obs_DF):
-    obs_DF['longitude'] = (obs_DF['longitude']+int(180)) % 360 - int(180)
+    obs_DF['longitude'] = (obs_DF['longitude']+180).mod(360) - 180
     return obs_DF
 
 

--- a/src/hdf5/cygnss_2ioda.py
+++ b/src/hdf5/cygnss_2ioda.py
@@ -218,7 +218,8 @@ def adjust_dateTime(obs_DF, dat_ref):
 
 
 def adjust_longitude(obs_DF):
-    obs_DF['longitude'] = (obs_DF['longitude']+180).mod(360) - 180
+    mask = obs_DF['longitude']>180
+    obs_DF.loc[mask, 'longitude'] = obs_DF['longitude'].loc[mask].values - 360
     return obs_DF
 
 

